src/Core__Math.res

```diff
@@ -44,9 +54,19 @@ module Int = {
 @val external log10: float => float = "Math.log10"
 @val external log2: float => float = "Math.log2"
 @val external min: (float, float) => float = "Math.min"
-@variadic @val external minMany: array<float> => float = "Math.min"
+@variadic @val external _minMany: array<float> => float = "Math.min"
+let minMany = arr =>
+  switch arr {
+  | [] => 0.0
+  | arr => _minMany(arr)
+  }
```

Comment:
I'm not sure the `float` versions ought to be overridden. `Math.min([])` does indeed return `0`, and `Math.min()` returns `Infinity`, but `Math.minMany([1, 2, 3])` maps to `Math.min(1, 2, 3)`, not `Math.min([1, 2, 3])`, and so it seems more consistent to have `Math.minMany([])` map to `Math.min()`, not `Math.min([])`.

These are also edge cases that are very unlikely to occur in practice, since variadic calls require syntactic arrays there's little chance of it happening by accident. I'd suggest the aim of this is restricted to just making the types sound.